prebid-mobile/modules/rendering/ios-sdk-integration-gam.md

```diff
@@ -48,7 +36,13 @@ To avoid this error add the following line to your app right after initializatio
 GAMUtils.shared.initializeGAM()
 ```
 
-## Banner API
+## AdUnit-Specific instructions
+
+This section covers integration details for different ad formats. In each scenario, you'll be asked for a `configId` - this is a key worked out with your Prebid Server provider. It's used at runtime to pull in the bidders and parameters specific to this adunit. Depending on your Prebid Server partner, it may be a UUID or constructed out of parts like an account number and adunit name.
+
+### Banners
+
+#### Display Banners
 
 Integration example:
```

Comment:
Need to choose and follow the same approach for introducing the code snippets. The code in iOS Bidding-Only doc has different formatting and include the language name: 

https://github.com/prebid/prebid.github.io/pull/5507/files#diff-960ae6a6923b911e5d13809fe793b5ec1909af0577b7c621c6aa88c4bd896fc5R14

We can address it in a separate scope because it can be challenging to find all the places in the PR.

Reply:
Someone needs to review all the snippets anyhow for correctness and good comments. How about we make the intros consistent at that time?

Reply:
I've creates a follow up ticket for this